apps/services/auth/ids-api/src/app/user-profile/user-profile.service.ts

```diff
@@ -45,7 +46,13 @@ export class UserProfileService {
     private companyRegistryApi: CompanyRegistryClientService,
     @Inject(LOGGER_PROVIDER)
     private logger: Logger,
-  ) {}
+    @Optional()
+    private alsoUseFakeApi: boolean,
```

Comment:
Ah sorry we need the `alsoUseFakeApi` member definition on the class, but we can skip assigning it a default value as the optional chaining in the constructor will handle that.

So add L#34 again as `private alsoUseFakeApi: boolean` and remove L#50

Also `@Optional()` decorator was probably meant for the `configService`?

Reply:
Fixed.